src/Support/Contracts/CacheKeyAndTags.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Contracts;
```

Comment:
I would rather have this in `Nuwave\Lighthouse\Cache`, trying to get away from having all interfaces lumped together.